sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -898,11 +899,15 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
       (1, 2), (1, 1), (2, 1), (2, 2))
   }
 
-  test("dropDuplicates should not change child plan output") {
-    val ds = Seq(("a", 1), ("a", 2), ("b", 1), ("a", 1)).toDS()
-    checkDataset(
-      ds.dropDuplicates("_1").select(ds("_1").as[String], ds("_2").as[Int]),
-      ("a", 1), ("b", 1))
+  test("SPARK-19065 dropDuplicates should not create expressions using the same id") {
```

Comment:
It's in my fist commit: https://github.com/apache/spark/pull/16564/commits/13f54a93c0cf31a38455e90aec722e890af980c6

I removed it because it's not a Structured Streaming issue.